cmd/security-secrets-setup/Dockerfile

```diff
@@ -80,13 +76,11 @@ COPY --from=build-env /edgex-go/cmd/security-secrets-setup/entrypoint.sh /usr/lo
 RUN chmod +x /usr/local/bin/entrypoint.sh \
     && ln -s /usr/local/bin/entrypoint.sh /
 
-# Create assets folder (needed for unseal key/s, root token and tmp)
-# Run CA/Vault and Kong PKI/TLS setups and peform housekeeping tasks
-RUN mkdir $BASE_DIR/config/assets && \
-    chown -R vault:vault $BASE_DIR && \
-    chmod 644 $BASE_DIR/config/local.hcl && \
-    chmod 744 security-secrets-setup
-
-VOLUME $BASE_DIR/config
+RUN chmod 755 security-secrets-setup
 
 ENTRYPOINT ["entrypoint.sh"]
+
+# after TLS assets are generated-
+# make edgex vault's TLS assets accessible by vault's user from vault service
+# from the implementation of vault service, it belongs to 100:1000 user group
+CMD ["sh", "-c", "${BASE_DIR}/security-secrets-setup generate ;chown -Rh 100:1000 /run/edgex/secrets/edgex-vault/"]
```

Comment:
Not that this is wrong, but Vault does it nicer:

https://github.com/hashicorp/docker-vault/blob/master/0.X/docker-entrypoint.sh

Vault would allow you to do:
```
command: "generate"
```
or
```
command: "security-secrets-setup generate"
```

and it would know what you mean.

If you supplied anything else like the above it would run it literally.